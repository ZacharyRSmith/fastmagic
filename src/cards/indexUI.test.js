import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import IndexUI from './indexUI';

it('renders without crashing', () => {
  const wrapper = shallow(
    <IndexUI {...{
    }} />
  );

  expect(toJson(wrapper)).toMatchSnapshot();
});
