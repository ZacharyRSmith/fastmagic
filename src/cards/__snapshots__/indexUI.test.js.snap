exports[`test renders without crashing 1`] = `
<div>
  <h1>
    MTG Cards
  </h1>
  <Search />
</div>
`;
